### ISSUE TITLE
Suggestion: Inform the user that changing the language requires a restart

### ISSUE THREAD

**Issue by HerrSersch** [ransome1/sleek#54]
Changing the language quits Sleek instantly. When I did this the first time I thought Sleek had silently crashed. I then assumed that it might just be automatically restarting, because tools oftentimes do this. After a while without anything happening I started Sleek manually again and switched languages back and forth to confirm that this is in fact intended and there is no crash.

Ideas how to improve the user feedback:

A) Add a sentence like "Changing the language requires a restart" to the language setting's description, possibly in bold. Disable automatic quitting.
B) Open an info popup that can be closed by pressing OK that reads something like "You have to restart Sleek for this change to take effect." Disable automatic quitting.
C) Open an info popup that reads something like "Sleek needs to restart to change the language. Restart now?", which can be answered with "Yes" and "No, I'll restart later". Yes would automatically restart, the other option closes the dialogue.

**Comment by ransome1** [comment]
@HerrSersch . Sleek is suppose to restart itself. I assume you are on 0.9.2 and you have downloaded a binary here on Github?

**Comment by HerrSersch** [comment]
Yes, I have tested this with 0.9.2 with the binaries from here.

I further looked into it and it seems it doesn't happen with all releases. The .exe does not automatically restart on Win7, neither does the .AppImage on Mint Cinnamon 20.1. The .deb does automatically restart though. I didn't notice the working .deb before because I often go for the .AppImage files first if they available.

I have the UAC turned off on Win7 in case that's relevant. I've also tried running it as Admin, but that did make no difference.

**Comment by ransome1** [comment]
That really is an odd behavior. But at least I now could replicate the problem too. I'll look into it.

**Comment by ransome1** [comment]
@HerrSersch . I double checked the implementation and it is done correctly according to the official Electron documentation.

Also I digged a little into it and at least for the AppImage it seems to be an issue within the Electron framework or Electron builder. Somebody posted a workaround (https://github.com/electron-userland/electron-builder/issues/1727), which I tried but it didn't work. Whenever it is fixed upstream, it will be fixed within sleek.

As for now there is nothing I can do about it as it is not a critical bug and it currently affects only the .exe and AppImage builds. For a temporary solution I will append a description text in the settings modal.
